2021-10-24 07:35:40.8593|INFO|Info|User Iserial:5
2021-10-24 07:35:40.9222|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":{"InnerException":null,"Message":"Cannot insert duplicate key row in object 'dbo.TblAdjustmentDetail' with unique index 'IX_TblAdjustmentDetail'. The duplicate key value is (151, 310961).\r\nThe statement has been terminated.","StackTrace":"   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()\r\n   at System.Data.SqlClient.SqlDataReader.get_MetaData()\r\n   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)\r\n   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)\r\n   at System.Data.Mapping.Update.Internal.DynamicUpdateCommand.Execute(UpdateTranslator translator, EntityConnection connection, Dictionary`2 identifierValues, List`1 generatedValues)\r\n   at System.Data.Mapping.Update.Internal.UpdateTranslator.Update(IEntityStateManager stateManager, IEntityAdapter adapter)","Type":"System.Data.SqlClient.SqlException"},"Message":"An error occurred while updating the entries. See the inner exception for details.","StackTrace":"   at CCWFM.Web.Service.WarehouseOp.WarehouseService.InsertRemainingImportedItems(Int32 headerIserial, List`1 importedList) in D:\\CCWFM\\CCWFM.Web\\Service\\WarehouseOp\\Adjustment.cs:line 530\r\n   at SyncInvokeInsertRemainingImportedItems(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.Data.UpdateException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"An error occurred while updating the entries. See the inner exception for details.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"An error occurred while updating the entries. See the inner exception for details."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.WarehouseServiceClientChannel.EndInsertRemainingImportedItems(IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.CCWFM.WarehouseService.WarehouseService.EndInsertRemainingImportedItems(IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.OnEndInsertRemainingImportedItems(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.WarehouseService.InsertRemainingImportedItemsCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.AdjustmentViewModel.<.ctor>b__12_27(Object s, InsertRemainingImportedItemsCompletedEventArgs e)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.OnInsertRemainingImportedItemsCompleted(Object state)"}
2021-10-24 07:35:41.6119|INFO|Info|User Iserial:5
2021-10-24 07:35:41.6119|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":{"InnerException":null,"Message":"Cannot insert duplicate key row in object 'dbo.TblAdjustmentDetail' with unique index 'IX_TblAdjustmentDetail'. The duplicate key value is (151, 311040).\r\nThe statement has been terminated.","StackTrace":"   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()\r\n   at System.Data.SqlClient.SqlDataReader.get_MetaData()\r\n   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)\r\n   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)\r\n   at System.Data.Mapping.Update.Internal.DynamicUpdateCommand.Execute(UpdateTranslator translator, EntityConnection connection, Dictionary`2 identifierValues, List`1 generatedValues)\r\n   at System.Data.Mapping.Update.Internal.UpdateTranslator.Update(IEntityStateManager stateManager, IEntityAdapter adapter)","Type":"System.Data.SqlClient.SqlException"},"Message":"An error occurred while updating the entries. See the inner exception for details.","StackTrace":"   at CCWFM.Web.Service.WarehouseOp.WarehouseService.InsertRemainingImportedItems(Int32 headerIserial, List`1 importedList) in D:\\CCWFM\\CCWFM.Web\\Service\\WarehouseOp\\Adjustment.cs:line 530\r\n   at SyncInvokeInsertRemainingImportedItems(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.Data.UpdateException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"An error occurred while updating the entries. See the inner exception for details.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"An error occurred while updating the entries. See the inner exception for details."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.WarehouseServiceClientChannel.EndInsertRemainingImportedItems(IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.CCWFM.WarehouseService.WarehouseService.EndInsertRemainingImportedItems(IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.OnEndInsertRemainingImportedItems(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.WarehouseService.InsertRemainingImportedItemsCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.AdjustmentViewModel.<.ctor>b__12_27(Object s, InsertRemainingImportedItemsCompletedEventArgs e)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.OnInsertRemainingImportedItemsCompleted(Object state)"}
2021-10-24 07:35:41.6189|INFO|Info|User Iserial:5
2021-10-24 07:35:41.6189|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":{"InnerException":null,"Message":"Cannot insert duplicate key row in object 'dbo.TblAdjustmentDetail' with unique index 'IX_TblAdjustmentDetail'. The duplicate key value is (151, 311110).\r\nThe statement has been terminated.","StackTrace":"   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()\r\n   at System.Data.SqlClient.SqlDataReader.get_MetaData()\r\n   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)\r\n   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)\r\n   at System.Data.Mapping.Update.Internal.DynamicUpdateCommand.Execute(UpdateTranslator translator, EntityConnection connection, Dictionary`2 identifierValues, List`1 generatedValues)\r\n   at System.Data.Mapping.Update.Internal.UpdateTranslator.Update(IEntityStateManager stateManager, IEntityAdapter adapter)","Type":"System.Data.SqlClient.SqlException"},"Message":"An error occurred while updating the entries. See the inner exception for details.","StackTrace":"   at CCWFM.Web.Service.WarehouseOp.WarehouseService.InsertRemainingImportedItems(Int32 headerIserial, List`1 importedList) in D:\\CCWFM\\CCWFM.Web\\Service\\WarehouseOp\\Adjustment.cs:line 530\r\n   at SyncInvokeInsertRemainingImportedItems(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.Data.UpdateException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"An error occurred while updating the entries. See the inner exception for details.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"An error occurred while updating the entries. See the inner exception for details."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.WarehouseServiceClientChannel.EndInsertRemainingImportedItems(IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.CCWFM.WarehouseService.WarehouseService.EndInsertRemainingImportedItems(IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.OnEndInsertRemainingImportedItems(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.WarehouseService.InsertRemainingImportedItemsCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.AdjustmentViewModel.<.ctor>b__12_27(Object s, InsertRemainingImportedItemsCompletedEventArgs e)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.OnInsertRemainingImportedItemsCompleted(Object state)"}
2021-10-24 07:35:41.6599|INFO|Info|User Iserial:5
2021-10-24 07:35:41.6599|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":{"InnerException":null,"Message":"Cannot insert duplicate key row in object 'dbo.TblAdjustmentDetail' with unique index 'IX_TblAdjustmentDetail'. The duplicate key value is (151, 311187).\r\nThe statement has been terminated.","StackTrace":"   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()\r\n   at System.Data.SqlClient.SqlDataReader.get_MetaData()\r\n   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)\r\n   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)\r\n   at System.Data.Mapping.Update.Internal.DynamicUpdateCommand.Execute(UpdateTranslator translator, EntityConnection connection, Dictionary`2 identifierValues, List`1 generatedValues)\r\n   at System.Data.Mapping.Update.Internal.UpdateTranslator.Update(IEntityStateManager stateManager, IEntityAdapter adapter)","Type":"System.Data.SqlClient.SqlException"},"Message":"An error occurred while updating the entries. See the inner exception for details.","StackTrace":"   at CCWFM.Web.Service.WarehouseOp.WarehouseService.InsertRemainingImportedItems(Int32 headerIserial, List`1 importedList) in D:\\CCWFM\\CCWFM.Web\\Service\\WarehouseOp\\Adjustment.cs:line 530\r\n   at SyncInvokeInsertRemainingImportedItems(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.Data.UpdateException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"An error occurred while updating the entries. See the inner exception for details.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"An error occurred while updating the entries. See the inner exception for details."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.WarehouseServiceClientChannel.EndInsertRemainingImportedItems(IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.CCWFM.WarehouseService.WarehouseService.EndInsertRemainingImportedItems(IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.OnEndInsertRemainingImportedItems(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.WarehouseService.InsertRemainingImportedItemsCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.AdjustmentViewModel.<.ctor>b__12_27(Object s, InsertRemainingImportedItemsCompletedEventArgs e)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.OnInsertRemainingImportedItemsCompleted(Object state)"}
2021-10-24 07:35:41.6678|INFO|Info|User Iserial:5
2021-10-24 07:35:41.6678|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":{"InnerException":null,"Message":"Cannot insert duplicate key row in object 'dbo.TblAdjustmentDetail' with unique index 'IX_TblAdjustmentDetail'. The duplicate key value is (151, 311259).\r\nThe statement has been terminated.","StackTrace":"   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()\r\n   at System.Data.SqlClient.SqlDataReader.get_MetaData()\r\n   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)\r\n   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)\r\n   at System.Data.Mapping.Update.Internal.DynamicUpdateCommand.Execute(UpdateTranslator translator, EntityConnection connection, Dictionary`2 identifierValues, List`1 generatedValues)\r\n   at System.Data.Mapping.Update.Internal.UpdateTranslator.Update(IEntityStateManager stateManager, IEntityAdapter adapter)","Type":"System.Data.SqlClient.SqlException"},"Message":"An error occurred while updating the entries. See the inner exception for details.","StackTrace":"   at CCWFM.Web.Service.WarehouseOp.WarehouseService.InsertRemainingImportedItems(Int32 headerIserial, List`1 importedList) in D:\\CCWFM\\CCWFM.Web\\Service\\WarehouseOp\\Adjustment.cs:line 530\r\n   at SyncInvokeInsertRemainingImportedItems(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.Data.UpdateException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"An error occurred while updating the entries. See the inner exception for details.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"An error occurred while updating the entries. See the inner exception for details."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.WarehouseServiceClientChannel.EndInsertRemainingImportedItems(IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.CCWFM.WarehouseService.WarehouseService.EndInsertRemainingImportedItems(IAsyncResult result)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.OnEndInsertRemainingImportedItems(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.WarehouseService.InsertRemainingImportedItemsCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.AdjustmentViewModel.<.ctor>b__12_27(Object s, InsertRemainingImportedItemsCompletedEventArgs e)\r\n   at CCWFM.WarehouseService.WarehouseServiceClient.OnInsertRemainingImportedItemsCompleted(Object state)"}
2021-10-24 13:44:21.7938|INFO|Info|User Iserial:1518
2021-10-24 13:44:21.8028|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/BankDepositOp/BankDepositService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   at System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   at System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   at System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   at System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   at System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.BankDepositServiceClientChannel.EndUpdateOrInsertCashDepositHeader(Int32& outindex, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.CCWFM.BankDepositService.BankDepositService.EndUpdateOrInsertCashDepositHeader(Int32& outindex, IAsyncResult result)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.OnEndUpdateOrInsertCashDepositHeader(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.BankDepositService.UpdateOrInsertCashDepositHeaderCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.Gl.CashDepositViewModel.<.ctor>b__6_33(Object s, UpdateOrInsertCashDepositHeaderCompletedEventArgs x)\r\n   at CCWFM.BankDepositService.BankDepositServiceClient.OnUpdateOrInsertCashDepositHeaderCompleted(Object state)"}
